lib/team_api/canonicalizer.rb

```diff
@@ -9,8 +9,12 @@ class Canonicalizer
     # Canonicalizes the order and names of certain fields within site_data.
     def self.canonicalize_data(site_data)
       sort_collections site_data
```

Comment:
also, in the case that there are multiple args and they span several lines, parens are a very helpful organizational tool.